q2_amr/card/tests/test_utils.py

```diff
@@ -198,3 +205,25 @@ def test_colorify(self):
         colored_string = colorify(string)
         expected_output = "\033[1;32mHello, world!\033[0m"
         self.assertEqual(colored_string, expected_output)
+
+    def test_copy_files(self):
+        with tempfile.TemporaryDirectory() as tmp:
+            # Create source files
+            file_path_1 = os.path.join(tmp, "file1.txt")
+            file_path_2 = os.path.join(tmp, "file2.txt")
+
+            with open(file_path_1, "w"), open(file_path_2, "w"):
+                pass
+
+            # Call the function
+            file_paths = [file_path_1, file_path_2]
```

Comment:
You have a hundred files in your `data` directories - why not just use some of those instead of this magic? Also, I think if your test class inherits from the TestPlugin class then it should have something like a `self.temp_dir` available.